packages/workflow/src/states.ts

```diff
@@ -34,16 +34,18 @@ export type RejectCorrection = Nominal<{}, 'RejectCorrection'>
 export type ApproveCorrection = Nominal<{}, 'ApproveCorrection'>
 export type MakeCorrection = Nominal<{}, 'MakeCorrection'>
 export type WaitForExternalValidation = Nominal<{}, 'WaitForExternalValidation'>
+export type Reject = Nominal<{}, 'Reject'>
 
 export type ActionIdentifiers = {
   REQUEST_CORRECTION: RequestCorrection
   APPROVE_CORRECTION: ApproveCorrection
   MAKE_CORRECTION: MakeCorrection
   REJECT_CORRECTION: RejectCorrection
   CERTIFY: Certify
-  VALIDATION: Validate
-  DECLARATION_UPDATED: Update
-  REGISTERED: Register
+  VALIDATE: Validate
+  DECLARATION_UPDATE: Update
```

Comment:
```suggestion
  UPDATE_DECLARATION: Update
```
makes more sense no?